vsts/pipelines/_buildParallel.yml

```diff
@@ -111,7 +108,9 @@ steps:
   inputs:
     type: InlineScript
     script: 'echo "success">$(Build.SourcesDirectory)/artifacts/$(Build.BuildNumber).txt'
-  condition: and(succeeded(), in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'))
+  condition: >
```

Comment:
this indentation is confusing. We could split the statements inside `and` one per line.